src/index.coffee

```diff
@@ -166,6 +166,19 @@ class PurecloudBot extends Adapter
 
       @realtime.sendMessage to, msg
 
+  reply: (envelope, messages...) ->
+    log 'robot', 'send', arguments...
+
+    log 'full_message', messages
+    for msg in messages
+      unless msg then continue
+      @robot.logger.debug "Sending to #{envelope.room or envelope.user?.id}: #{msg}"
+      log 'robot', 'send', envelope.user.id
```

Comment:
you probably want a ? on user here as well since you have it everywhere else in your function. 

Switch your `log`s to `debug` or just delete them.